Gems/WarehouseAssets/README.md

```diff
@@ -0,0 +1,138 @@
+## Warehouse Assets Gem ##
```

Comment:
nit: You don't need the trailing `###` (heading indicators), it's sufficient to just write `### Heading`